app/views/comments/_voting.html.erb

```diff
@@ -0,0 +1,9 @@
+<div class='voting'>
+<%= link_to '<span class="glyphicon glyphicon-arrow-up"></span>'.html_safe, post_upvote_path(post), :method => :post %>
+<strong><%= comment.content.inject(0) {|sum, v| sum + v.value } %></strong>
```

Comment:
Next, let's change this so that we can add a vote to the comment in question by changing line 3 to this:
```
<strong><%= comment.votes.inject(0) {|sum, v| sum + v.value } %></strong>
```